packages/lesswrong/lib/collections/ckEditorUserSessions/schema.ts

```diff
@@ -0,0 +1,22 @@
+import { accessFilterSingle, resolverOnlyField, schemaDefaultValue } from "../../utils/schemaUtils";
+import { userOwns } from "../../vulcan-users/permissions";
+
+const schema: SchemaType<DbCkEditorUserSession> = {
+  documentId: {
+    type: String,
+    nullable: false,
+    canRead: ['admins'],
+  },
+  userId: {
+    type: String,
+    nullable: false,
+    canRead: ['admins'], 
+  },
+  endedAt: {
```

Comment:
I am very confused why you don't also have `startedAt`

Reply:
Correct. Robert asked me to just use createdAt and not have a startedAt